e2e_tests/tests/cluster/test_users.py

```diff
@@ -74,6 +74,8 @@ def det_run(args: List[str]) -> str:
 
 
 def log_in_user(credentials: authentication.Credentials, expectedStatus: int = 0) -> None:
+    if expectedStatus == 0:
+        return api_utils.log_in_user_cli(credentials)
     username, password = credentials
     child = det_spawn(["user", "login", username])
     child.setecho(True)
```

Comment:
I think it's going to be really non-obvious from reading the calling code what this function is going to do.

Can you separate this into:
- `log_in_user(credentials)` that uses the python sdk.  This would be the standard usage.
- `log_in_user_cli(credentials, expected_status)` that explicitly uses the cli and might expect a nonzero status.

The way it's currently written, I think it's really sneaky that this function sometimes uses the cli and sometimes uses the sdk.

Reply:
yeah, I agree I was just trying to avoid having to change more code considering this is used in a bunch of places but it shouldn't be too bad to update.

Reply:
updated.